src/Buffer.mo

```diff
@@ -1759,6 +2451,20 @@ module {
   /// have less than `size` elements if the number of elements is not divisible
   /// by the chunk size.
   ///
+  /// Example:
+  /// ```motoko include=initialize
+  ///
+  /// let buffer = Buffer.Buffer<Nat>(2);
+  /// buffer.add(1);
+  /// buffer.add(2);
+  /// buffer.add(3);
+  /// buffer.add(4);
+  /// buffer.add(5);
+  /// buffer.add(6);
+  ///
+  /// Buffer.chunk<Nat>(buffer, 3);
```

Comment:
So this returns a Buffer<Buffer<Nat>> which doesn't display nicely by the interpreter in the docs page.

```suggestion
  /// import Nat "mo:base/Nat";
  ///
  /// let buffer = Buffer.Buffer<Nat>(2);
  /// buffer.add(1);
  /// buffer.add(2);
  /// buffer.add(3);
  /// buffer.add(4);
  /// buffer.add(5);
  /// buffer.add(6);
  ///
  /// let chunks = Buffer.chunk<Nat>(buffer, 3);
  /// Buffer.toText<Buffer.Buffer<Nat>>(chunks, func buf = Buffer.toText(buf, Nat.toText))
```